tests/ship_log.cpp

```diff
@@ -193,21 +248,8 @@ BOOST_DATA_TEST_CASE(basic_prune_test, bdata::xrange(2) * bdata::xrange(2) * bda
       t.check_not_present(7);
    });
 
-   //undo past the first stored
```

Comment:
I've left this test removed because the current log does not allow such behavior,
https://github.com/AntelopeIO/spring/blob/8c597cdc7aba4f2cb878664fc0fdc5375450840e/libraries/state_history/include/eosio/state_history/log.hpp#L224
I'm not confident trying to restore this behavior at the moment. It seems rather pathological though.